addon/index.js

```diff
@@ -117,6 +117,12 @@ export class QueryParamAwarePretender extends Pretender {
 
       let { result, message, handler } = matchers.get(search);
 
+      const qps = {};
+
+      for (const [key, value] of searchParams.entries()) {
+        qps[key] = value;
+      }
+
```

Comment:
Why do we have to do this rather than pass in a URL with query params to `super._handlerFor` and let pretender set the query params?

Can we express the difference in a test?

Reply:
@hjdivad as per @rwjblue comment,  looks like we intentionally do not pass the query params into the super._handlerFor call.

Reply:
updated the code to pass in the search string to the `super._handlerFor`